gtests/net/mptcp/sockopts/sockopt_cork_nodelay.pkt

```diff
@@ -0,0 +1,50 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
+
++0.0  socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0  setsockopt(3, SOL_TCP, TCP_CORK, [1], 4) = 0
+
++0    bind(3, ..., ...) = 0
++0    listen(3, 1) = 0
+
++0.0    <  addr[caddr0] > addr[saddr0]  S   0:0(0)               win 65535  <mss 1460, sackOK, TS val 4074410674 ecr 0,          nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
```

Comment:
if you don't need to use multiple subflows, you can remove `addr[caddr0] > addr[saddr0]` (and spaces below), that will be easier to read :)